Voronoi_diagram_2/include/CGAL/Delaunay_triangulation_on_sphere_adaptation_traits_2.h

```diff
@@ -39,22 +41,19 @@ struct Delaunay_triangulation_on_sphere_adaptation_traits_2
   typedef CGAL::Tag_false                                            Has_nearest_site_2;
   typedef CGAL_VORONOI_DIAGRAM_2_INS::Null_functor                   Nearest_site_2;
 
-  Delaunay_triangulation_on_sphere_adaptation_traits_2(const DTOS& dtos) : dtos(dtos) { }
+  Delaunay_triangulation_on_sphere_adaptation_traits_2(const Geom_traits& gt) : gt(gt) { }
 
   Access_site_2 access_site_2_object() const
-  { return Access_site_2(dtos); }
+  { return Access_site_2(gt); }
 
   Construct_Voronoi_point_2 construct_Voronoi_point_2_object() const
-  { return Construct_Voronoi_point_2(dtos); }
+  { return Construct_Voronoi_point_2(gt); }
 
   Nearest_site_2 nearest_site_2_object() const
   { return Nearest_site_2(); }
 
-  typedef typename Geom_traits::Point_on_sphere_2                    Point_2;
-  typedef Point_2                                                    Site_2;
-
 private:
-  const DTOS& dtos;
+  const Geom_traits gt; // intentional copy
```

Comment:
Your comment should explain why that is intentional, for readers to understand.

Reply:
The detail is in the commit message.